README.md

```diff
@@ -17,3 +17,13 @@ Real-world use cases for Kōjō and instructions for getting Kōjō up and runni
 XDebug version greater than `xdebug-2.7.0alpha1` is required when trying to debug Kōjō. This version of XDebug resolves issues (https://bugs.xdebug.org/938) caused by the way the Kōjō forks using `pcntl`.
 
 If you are using PhpStorm and you have more concurrent Kōjō jobs running than the `Max. simultaneous connections` defined for your XDebug listener, Kōjō will appear to hang. To avoid this, increase your `Max. simultaneous connections` to the max value of `20` in PhpStorm's preferences under `Languages & Frameworks` > `PHP` > `Debug`, `External connections` section.
+
+## Kojo Hanging Doing Nothing?
```

Comment:
```suggestion
## Kōjō appears to be hanging during local development?
```

It's All abōut thë speciał letterš here in Kōjō land. Anything else would be a Kōjō nønø.